paramnb/__init__.py

```diff
@@ -54,6 +52,14 @@ def HTMLWidget(*args, **kw):
     return ipywidgets.HTML(*args,**kw)
 
 
+def ListSelectorWidget(*args, **kw):
+    """Forces a parameter value to be text, displayed as HTML"""
+    if len(kw['options']) > 20:
+        return CrossSelect(*args, **kw)
```

Comment:
The magic number 20 should definitely be configurable.  How about a class parameter of the CrossSelect class?  I know it's not actually used there, but from a user perspective that seems like a reasonable place to be setting it (as functions can't ordinarily have Parameters). Or I guess this can be made into a ParameterizedFunction, solely to be able to store that parameter.  Either way, hardcoding 20 isn't ok.

There should also be some documented values to force it never to be used and always to be used, presumably -1 for "always" and I guess None for never?